monitor/plugins/kubejobs/plugin.py

```diff
@@ -156,3 +159,24 @@ def monitoring_application(self):
 
             self.LOG.log(ex.message)
             raise
+
+    def validate(self, data):
+        data_model = {
+            "datasource_type": six.string_types,
+            "enable_visualizer": bool,
+            "expected_time": int,
+            "number_of_jobs": int,
+            "redis_ip": six.string_types,
+            "redis_port": int,
+            "submission_time": six.string_types
+        }
+
+        for key in data_model:
+            if (key not in data):
+                print(key)
+                raise ex.BadRequestException(
+                    "Variable \"{}\" is missing".format(key))
+
+            if (not isinstance(data[key], data_model[key])):
+                raise ex.BadRequestException(
+                    "\"{}\" has unexpected variable type".format(key))
```

Comment:
include expected and actual types in the message as well, for better user experience.